composer.json

```diff
@@ -33,7 +33,7 @@
         "composer-exit-on-patch-failure": true,
         "patches": {
             "drupal/core": {
-                "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2599228-31-tests-only.patch"
+                "https://www.drupal.org/project/drupal/issues/2599228": "https://www.drupal.org/files/issues/2018-05-17/2599228-51.patch"
```

Comment:
This is changing the patches, this might affect other components that are depending on this. If composer-patches encounters two different versions for the same patch in the dependency tree then it will throw an error.

I would suggest that we wait with this until we were able to remove the patches.

Reply:
Since we are going to deal with the patches in a follow up I'm approving this PR.